services/gundeck/src/Gundeck/Push/Native.hs

```diff
@@ -57,12 +57,17 @@ push1 m a = do
             view monitor >>= counterIncr (path "push.native.errors")
     return r
   where
-    -- * Transient notifications must be delivered "now or never".
-    -- * Others use the default level of the specific platforms, which is 4 weeks
-    --   for both APNS* and GCM
-    ttl = if msgTransient m
-          then Just (Aws.Seconds 0)
-          else Nothing
+    -- TODO: REFACTOR: this smells like a bug that we've dragged along for a while: according to
+    -- current logic (previous to this PR), 'Notice' is never transient.  but we have sent nothing
+    -- but 'Notice' native notifications for a while.  have we sent out native notifications for
+    -- transient messages at all?  that may have led to clients being woken up, pulling the event
```

Comment:
>  have we sent out native notifications for transient messages at all? 

This used to happen when we used cipher text messages; in such cases, clients could simply decrypt the symmetrically encrypted message and be done with it (didn't have to ping the backend).

>but transient 'Notification's are translated to 'Notice'

The idea is that transient messages must be delivered _now or never_, thus the '0' TTL (e.g., the device is off). Seems to me now that we're changing the current behaviour, are we not?